e2e/peer_ops_test.go

```diff
@@ -24,12 +29,42 @@ func TestAddRemovePeer(t *testing.T) {
 	defer g2.EraseWorkdir()
 	r.True(g2.IsRunning())
 
+	g3, err := spawnGlusterd("./config/3.yaml", true)
+	r.Nil(err)
+	defer g3.Stop()
+	defer g3.EraseWorkdir()
+	r.True(g3.IsRunning())
+
 	// add peer: ask g1 to add g2 as peer
 	reqBody := strings.NewReader(fmt.Sprintf(`{"addresses": ["%s"]}`, g2.PeerAddress))
 	resp, err := http.Post("http://"+g1.ClientAddress+"/v1/peers", "application/json", reqBody)
 	r.Nil(err)
 	defer resp.Body.Close()
-	r.Equal(resp.StatusCode, 201)
+	r.Equal(http.StatusCreated, resp.StatusCode)
+
+	time.Sleep(5 * time.Second)
```

Comment:
This is just to allow elasticetcd to properly form the etcd cluster. We need to add a cleaner method to detect this, and once we do that, these sleeps can be removed.